octoprint_mrbeam/templates/hard_refresh_overlay.jinja2

```diff
@@ -0,0 +1,23 @@
+<div id="hard_refresh_overlay" class="modal hide fade" data-backdrop="static">
+    <div class="modal-header">
+        <h3><i class="icon icon-warning-sign"></i> {{ _('Refresh is needed after the last Software Update') }}</h3>
+    </div>
+    <div class="modal-body">
+        <div class="row-fluid">
+            <div class="span5 image_description">
+                <p>{{ _("An update has been installed on this Mr Beam. To use the new functions, the user interace must be completely reloaded.") }}</p>
+                <p> {{ _("We call this a hard refresh. To do this, press the three keys at the same time, depending on your operating system, as shown in the graphic:") }}</p>
+            </div>
```

Comment:
> @Josef-MrBeam this picture shows "Strg", what about English version with CTRL ? we will start selling in the UK soon octoprint_mrbeam/static/img/windows_refresh.png

I discussed with Daniel and we said it will be enough if we translate the Text over the images to show the correct name.

Reply:
ok, thanks for the hint